### DISCUSSION TITLE
JSON feed validator comes back with timeout error

### DISCUSSION THREAD

**Discussion by ginageick** [code4recovery/12-step-meeting-list#1392] [General]
Hi there,
I had moved the area's wp installation to a different folder - all good and well. But, the JSON feed (in its new location) now does not validate. cURL 28 error (time out). The feed is here: https://area84aa.org/aa/wp-admin/admin-ajax.php?action=meetings
This isn't a problem until there's a new meeting or any meeting changes. The App, right now, functions as expected.
I've also posed this question to our server support team - but figured I'd ask you fine folks as well, for good measure. Any ideas?
Thank you!
Gina G.
Area 84 Northeast Ontario

**Comment by SIA-WEBSUP** [comment]
i would venture to guess that if the validator isn't able to pull this feed then the meeting guide app writ large is also unable to pull the feed. meaning the meeting guide app is showing you whatever information it was last able to pull for your area, which is why 'new meetings' and 'updates' aren't showing up on the meeting guide app.

was your feed previously on the exact website and sever, just one level up?
and when it was there, did the meeting guide app ( and the validator ) able to pull the feed successfully?

**Comment by ginageick** [comment]
Yep, everything was one directory down, actually. But yes, the feed validated then and is not, now. Because it is a timeout error, I cannot really do anything from my end (my server).

**Reply by SIA-WEBSUP** [reply]
Hmmm... actually I hate to toss the ball 'in your court', but this is likely an issue you'll need to solve in conjunction with your hosting service provider. 

It isn't an issue with the plugin and I doubt the meeting guide app can solve for you. Although you could check with appsupport@aa.org to see if other aa entities have run into a similar issue.

The most likely explanation is that your provider is blocking the meeting guide app from pulling your feed. 
I suggest you check with your provider to see if they've 'denylisted' meetingguide.org. Or if they can 'allowlist' meetingguide.org.

**Comment by ginageick** [comment]
Well, I went through every security app/firewall and there is no block. The ping address for meetingguide.org is now whitelisted everywhere. Makes no difference. Still not validating.  :(

**Reply by SIA-WEBSUP** [reply]
Sorry to hear that this is still happening. Your host provider's customer support might have a solution for you.

Two other things you might consider pursuing:
1. get access the 'meeting guide entity dashboard' for your site if you don't already have it.  You can contact appsupport@aa.org to get this. find out more here: https://meetingguide.helpdocs.io/article/tevkz2ge05-meeting-guide-entity-dashboard

While you're at it, ask them if they have any suggestions for you. It may be that other sites have run into a similar issue.

2. this is way out in left field, but it's just how my troubleshooting brain works... i created a script on my intergroup's server that simply reads your json feed and spits it back out. you could ask the meeting guide app to use this feed for your area temporarily. just as a test. if it works, YOU'LL KNOW FOR CERTAIN that it's a network issue specific to your config. This would just be a temporary test.

Here is the URL to that script: https://suffolkny-aa.org/area48aa.php

**Reply by SIA-WEBSUP** [reply]
I assume this issue has been resolved since https://area84aa.org/aa/wp-admin/admin-ajax.php?action=meetings runs through the meeting guide's JSON validator with no issues.

I'm going to go ahead and delete https://suffolkny-aa.org/area48aa.php.
